api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/AlgorithmConfigurations.scala

```diff
@@ -97,6 +100,10 @@ class AlgorithmConfigurations extends MultipleAlgorithmsConfiguration {
     val gamFields = Seq(ignoredCols, betaConstraints, gamCols)
     val gbmFields = Seq(monotonicity, calibrationDataFrame, ignoredCols)
     val drfFields = Seq(calibrationDataFrame, ignoredCols)
+    val upliftDrfFields = Seq(
+      ExplicitField("treatment_column", "HasTreatmentCol", "treatment"),
```

Comment:
Why do we need an explicit field for treatment? Could we just add another rule to  `ParameterNameConverter`?

Reply:
Yes, for the uplift algorithm, the new treatment column is crucial.  However I am not sure how this algorithm configuration works, so I am not sure if this is the correct way to add the treatment column.